go.mod

```diff
@@ -2,7 +2,8 @@ module github.com/tektoncd/operator
 
 require (
 	github.com/go-logr/logr v0.1.0
-	github.com/jcrossley3/manifestival v0.0.0-20191210210340-41c7c3a22365
+	github.com/manifestival/controller-runtime-client v0.2.0
+	github.com/manifestival/manifestival v0.5.0
```

Comment:
There is a bug in v0.5.0, the `namespace` will not be remove when deleting:
https://github.com/manifestival/manifestival/issues/61

I think the fix will be merge soon, and after that, we could upgrade the version.